project/app/api/predict.py

```diff
@@ -161,6 +174,24 @@ async def predict_gas(item: GasItem):
 
     return round(total, 2)
 
+@router.post('/predict/airbnb')
```

Comment:
if you add a tag to predictions here, it'll look better on the docks page.  like this:
`@router.post('/predict/airbnb', tags = ['Predictions'])